README.md

```diff
@@ -182,6 +182,51 @@ If one of the allocators starts setting the withdraw queue and/or supply queue t
 
 ## Getting Started
 
+### Package installation
+
+```bash
+npm install @morpho-org/metamorpho
+```
+
+```bash
+yarn add @morpho-org/metamorpho
+```
+
+### Usage
+
+Bundle a supply cap raise and a reallocation to the market:
+
+```typescript
+import { MetaMorphoAction } from "@morpho-org/metamorpho";
+
+const marketParams1 = {
+  collateralToken: "0x...",
+  loanToken: "0x...",
+  irm: "0x...",
+  oracle: "0x...",
+  lltv: 86_0000000000000000n,
+};
+
+const marketParams2 = {
+  collateralToken: "0x...",
+  loanToken: marketParams1.loanToken,
+  irm: "0x...",
+  oracle: "0x...",
+  lltv: 96_5000000000000000n,
+};
+
+await metamorpho.connect(curator).multicall([
+  MetaMorphoAction.acceptCap(marketParams2),
+  MetaMorphoAction.setSupplyQueue([id1, id2, id3]),
```

Comment:
Where are the `id`s defined ?
It might be a bit confusing.

Moreover the `setSupplyQueue` operation is not mentioned here:
> Bundle a supply cap raise and a reallocation to the market:

I think we should either mention the `setSupplyQueue` operation in the previous description (and show how to get the "id"s from the market params) or remove it.